### ISSUE TITLE
[BITV] 9.1.4.3/0.1 - Note: For text (including in focused state) under 18 points (or for bold text under 14 points), the contrast ratio between the foreground and background colors should be at least 4.5:1.  (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36948]
For large text, the contrast ratio should be 3:1 or greater.

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v0-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
I will go through views and components to check this contrast issues

**Comment by JuliaKirschenheuter** [comment]
Dear @michaelnissenbaum,

could you please give me some examples, what should be fixed in that issue? I've give through different views and components and can't find which one have  to be fixed ;(

Here are examples which already have a contrast 4.54:1

![Screenshot from 2023-08-14 13-08-32](https://github.com/nextcloud/server/assets/6078378/b1b2624a-1040-48af-99e3-04bf8d107ff5)
![Screenshot from 2023-08-14 12-04-04](https://github.com/nextcloud/server/assets/6078378/7526c945-bf4c-4228-8e64-76aa7333af7d)
![Screenshot from 2023-08-14 12-01-35](https://github.com/nextcloud/server/assets/6078378/de22475a-d26b-4a0a-9df0-4784c953f984)
![Screenshot from 2023-08-14 11-56-08](https://github.com/nextcloud/server/assets/6078378/af8aa174-a6f2-4cf7-8960-d60e371e21a4)
![Screenshot from 2023-08-14 11-55-04](https://github.com/nextcloud/server/assets/6078378/8c54d1f3-2f4d-40c4-a736-1af81944dbc8)
![Screenshot from 2023-08-14 11-54-46](https://github.com/nextcloud/server/assets/6078378/bd410376-cfbc-43e1-af4d-385c4232d946)
![Screenshot from 2023-08-14 11-54-15](https://github.com/nextcloud/server/assets/6078378/9d25607d-42c1-4c93-b5dd-fbd19b699324)
![Screenshot from 2023-08-14 11-53-53](https://github.com/nextcloud/server/assets/6078378/91eedb9e-1895-4341-a426-f12accbcb957)


What have to be done: color for the button "More activities .." in dark mode have to be changed:
![Screenshot from 2023-08-14 13-16-57](https://github.com/nextcloud/server/assets/6078378/dc823514-ad73-4a9d-978b-41b122ee5d0f)

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter "9.1.4.3/0.1" is just a note indicating which contrast criteria need to be met. I believe you can close this ticket. The specific issues are described separately for each page.

![CleanShot 2023-08-14 at 14 15 13@2x](https://github.com/nextcloud/server/assets/48948959/c76201de-81c9-4807-89ac-dd7298dc7588)

**Comment by JuliaKirschenheuter** [comment]
closing as detailed tickets will be fixed separately (specific issues are described separately for each page)
